src/hotel_reservations/mlflow_processor.py

```diff
@@ -67,16 +86,44 @@ def __init__(self, model):
 
             def predict(self, context, model_input):
                 if isinstance(model_input, pd.DataFrame):
-                    predictions = self.model.predict(X_test)
+                    predictions = self.model.predict(model_input)
                     predictions = {"Prediction": adjust_predictions(predictions[0])}
                     return predictions
                 else:
                     raise ValueError("Input must be a pandas DataFrame.")
 
-        self.wrapped_model = HousePriceModelWrapper(self.model)
-        self.example_prediction = self.wrapped_model.predict(context=None, model_input=X_test.iloc[0:1])
+        wrapped_model = HousePriceModelWrapper(self.model)
```

Comment:
it would be nicer if that wrapper would be defined somewhere else in a separate module and not under function